### ISSUE TITLE
Provide contacts to help with accessibility

### ISSUE THREAD

**Issue by ztolley** [uktrade/trade-elements#19]
As a component or pattern developer I would like to know handy contacts or communication methods (slack) to get help or advice with regards to making sure something produced will be accessible to all.
